### ISSUE TITLE
fresh install OH3, samba & file permissions

### ISSUE THREAD

**Issue by pspa** [openhab/openhabian#1346]
I encountered an unexpected file permission error on my network mapped shares: 

- had a few first attempts using OH3 and used option 50 to create a backup zip file. 
- (messed up my system trying to install squeezebox server) 
- fresh install of OH3 (OH 3.0.0)
- used option 14 to not only see, but also get access to my network mapped shares
- created an empty  "fresh install" backup using option 50, in order to create the backup folder (would be more logical if it is always there but empty or only populated with readme.txt)
- wanted to copy-paste my backup zip file into place but encountered a file permission error
- used option 14 again to fix file permission errors
- copy-pasted the backup zip file, no file permission errors now
- used option 51 to restore my setup (successfully) 

I did not expect the file permission errors, is this intentional, a technical hurdle or a bug?

thanks for the great openhabian tool

**Comment by mstormi** [comment]
_What_ error did you get on _which_ action?

Provide a debug log please else we can't help.

**Comment by pspa** [comment]
no real debug log to provide:
I have mapped the openhab user folder on my windows computer.   After a fresh install, I can see the folder fine but I cannot copy anything into the folder: windows give me an error window stating "you do not have permission to write into this folder" 
Unfortunately, I do not have the exact error phrase and recreating would require me to start a clean install again.  I don't have a spare  SD-card so I cannot test but I think I got the same error as this old [post](https://community.openhab.org/t/cant-access-folders-through-windows-explorer/53636)
Since it was an old post I did not want to bump it again.  I'm not facing an issue currently because rerunning option 14 fixed the error.
